types/src/errors/runtime_error.rs

```diff
@@ -477,7 +477,7 @@ pub enum UnlockFailureReason {
 enum FailureReasonID {
     ScriptShouldBeBurnt = 1u8,
     ScriptShouldNotBeBurnt = 2u8,
-    ScriptError = 3u8,
+    ErrorForScript = 3u8,
```

Comment:
I changed a variant name 'ScriptError' to 'ErrorForScript' in enum. In the enum, all variants have the same prefix: `Script`, so an error occurred. But if subtract the prefix from all names, we cannot identify the role of the variable properly. So, only 'ScriptError' was changed; The prefix error is solved.

Reply:
You mean you met the Clippy error ["enum_variant_names"](https://rust-lang.github.io/rust-clippy/master/index.html#enum_variant_names)?

How about changing the name of the enum from `FailureReasonID` to `ScriptFailureReasonID`?

Reply:
```rust
enum ScriptFailureReasonID {
    ShouldBeBurnt = 1u8,
    ShouldNotBeBurnt = 2u8,
    Error = 3u8,
}
```
Is it right?

Reply:
Good!